### ISSUE TITLE
How to set styles for the shadow-root?

### ISSUE THREAD

**Issue by brechtm** [2A5F/shadow#97]
It's not clear to me how I can define styles for a shadow-root. It would be great if the documentation could cover this use-case.

**Comment by 2A5F** [comment]
You can use like
https://www.npmjs.com/package/vc-style
https://www.npmjs.com/package/vue-component-style

**Comment by brechtm** [comment]
Yes, I've managed by adding a `<link>` element inside the shadow root. It's not clear to me how using vc-style would work.

I was hoping I would be able to instantiate a component inside a shadow root and have it's `<style>` applied inside the shadow root, but alas, that doesn't work. This package's documentation only covers the bare minimum at the moment and leaves this kind of stuff up to the user to figure out. This is especially challenging if one is not familiar with shadow DOM. It would be great if the docs would provide examples of use cases that are supported.

**Comment by 2A5F** [comment]
I don't know how to solve this (how to provide the style), but if anyone can, welcome pr

**Comment by sam585456525** [comment]
> I don't know how to solve this (how to provide the style), but if anyone can, welcome pr
I have made a pr to solve this problem, please take some tome to  review.
#255
